torchquad/integration/vegas_stratification.py

```diff
@@ -6,22 +6,108 @@
 
 
 class VEGASStratification:
-    """The stratification used for VEGAS Enhanced. Refer to https://arxiv.org/abs/2009.05112."""
+    """The stratification used for VEGAS Enhanced. Refer to https://arxiv.org/abs/2009.05112.
+    Implementation inspired by https://github.com/ycwu1030/CIGAR/ .
+    EQ refers to equation in the paper.
+    """
 
-    def __init__(self) -> None:
-        pass
+    def __init__(self, dim=1, N_strat=10, beta=0.75):
+        """Initialize the VEGAS stratification
 
-    def strat_accumulate_weight(self, idx, weight):
-        pass
+        Args:
+            dim (int, optional): Dimensionality. Defaults to 1.
+            N_strat (int, optional): Number of stratification steps per dim. Defaults to 10.
+            beta (float, optional): Beta parameter from VEGAS Enhanced. Defaults to 0.75.
+        """
+        self.dim = dim
+        self.N_strat = N_strat  # stratification steps per dim
+        self.beta = beta  # variable controlling adaptiveness in stratification 0 to 1
+        self.N_cubes = self.N_strat ** self.dim  # total number of subdomains
+        self.V_cubes = (1.0 / self.N_strat) ** self.dim  # volume of hypercubes
```

Comment:
Pablo's comment for himself: In the paper they have 
![bilde](https://user-images.githubusercontent.com/70584234/115861295-edceb880-a432-11eb-9e86-cc414719d000.png)

Reply:
Implemented as in the paper now.